src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You can use a single `Lottery` instance to generate multiple balls. This will reduce the memory footprint and the overall complexity of your code.